developing/nix/README.md

```diff
@@ -162,86 +162,32 @@ nix-env -e chamber
 
 ### Profiles
 
-Profiles can be used at a global level, on a per-project basis, or
-on a per-repo basis.
-
-E.g. for MilMove, you might want a profile that contains things like
-`aws-vault`, `chamber`, etc. which spans more than just the `mymove`
-repo. On the other hand, you may not want to use a global profile
-(like the default one mentioned in the next section) because you may
-need different versions of `aws-vault` for different projects.
+Profiles can be used at a global level and on a per-repo basis.
 
 #### Default
 
 When it's installed, `nix` creates a profile which will store all the
-packages you install by default, unless you tell it to use a different
-profile when you're installing a package, or you switch the active
-profile to be another one.
+packages you install by default. It will initially only contain the
+packages needed for `nix` itself to work properly.
+
+If you want, you could tell it to use a different profile. Either on a
+one-off command or by switching the active profile. This is covered
+more in the [Profiles: Advanced section](#profiles:-advanced).
 
 Default profile is in: `/nix/var/nix/profiles/per-user/<username>/profile/`
 E.g. `/nix/var/nix/profiles/per-user/felipe/profile/`
 
-#### Creating a Profile
-
-You can create a new profile wherever you want, though convention seems
-to be to do so in
-`/nix/var/nix/profiles/`
-or
-`/nix/var/nix/profiles/per-user/<username>/`
-
-To create a new profile, you can run the command below. Note that this
-won't actually set this new profile as the active one, that's in the
-next section.
-
-```shell
-nix-env -p /nix/var/nix/profiles/<profile name> -i nix nss-cacert
-```
+#### Repo Profiles
 
-E.g.
+Ideally you'll have a profile defined at a repo level that will house
+your repo's dependencies. Then you can pre-pend this repo profile to
+your `PATH` when you're working with the repo, while keeping your
+default profile in the `PATH` behind the repo profile to still have
+access to `nix` commands. See
+[Working with Existing Nix Expressions](#working-with-existing-nix-expressions)
+for more details on how to set this up.
```

Comment:
I like this! How would you feel about documenting the `NIX_PROFILE` environment variable here?

Reply:
Ah good point, I'll add that.